### PR TITLE
fix: publishing jobs

### PR DESCRIPTION
**Pull request by leogr**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> If contributing rules or changes to rules, please make sure to also uncomment one of the following line:

> /kind rule-update

> /kind rule-create

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area build

> /area engine

> /area rules

> /area tests

> /area proposals

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
Follows up #1577 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -680,7 +680,7 @@ workflows:
             branches:
               only: master
           requires:
-            - "tests/integration-static"
+            - "build/centos7"
```
Comment (leodido) [inline .circleci/config.yml:683]:
```suggestion
            - "tests/integration"
```

**Review by leodido — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (leogr) [reply .circleci/config.yml]: Done! Tnx!

**Review by fntlnz — APPROVED** [review]

**Pull request merged**